src/material-experimental/mdc-button/fab.ts

```diff
@@ -39,8 +39,11 @@ import {ThemePalette} from '@angular/material-experimental/mdc-core';
   selector: `button[mat-fab], button[mat-mini-fab]`,
   templateUrl: 'button.html',
   styleUrls: ['fab.css'],
-  inputs: MAT_BUTTON_INPUTS,
-  host: MAT_BUTTON_HOST,
+  inputs: [...MAT_BUTTON_INPUTS, 'extended'],
```

Comment:
If I remember correctly, the spread operator inside metadata isn't support by ViewEngine so we'll have to list all of these out individually.